src/sundials/sundials_adaptcontroller.c

```diff
@@ -64,23 +64,6 @@ SUNAdaptController SUNAdaptController_NewEmpty(SUNContext sunctx)
   return (C);
 }
 
-/* -----------------------------------------------------------------
- * Free a generic SUNAdaptController (assumes content is already empty)
- * ----------------------------------------------------------------- */
-
-void SUNAdaptController_DestroyEmpty(SUNAdaptController C)
-{
-  if (C == NULL) { return; }
-
-  /* free non-NULL ops structure */
-  if (C->ops) { free(C->ops); }
-  C->ops = NULL;
-
-  /* free overall SUNAdaptController object and return */
-  free(C);
-  return;
-}
```

Comment:
I think the intent was for this to be public but it's missing from the header

Reply:
I agree with @gardner48.  If I recall correctly, I'd included it as a utility for user-provided `SUNAdaptController` implementations (like `SUNAdaptController_NewEmpty`).  We should add it to the header.